openedx/core/djangoapps/content/course_overviews/signals.py

```diff
@@ -88,21 +113,46 @@ def _check_for_course_date_changes(previous_course_overview, updated_course_over
         )
 
 
-def _log_start_date_change(previous_course_overview, updated_course_overview):  # lint-amnesty, pylint: disable=missing-function-docstring
+def _log_start_date_change(previous_course_overview, updated_course_overview):
+    """
+    Utility function to log a course run's start date when updating a course overview. This log only appears when the
```

Comment:
I know you're only adding a comment here, so this is just a reaction, but what an odd utility function.  I assume that 7 years ago when it was written it solved a specific and weird problem,  but if we were re-factoring this writ large I'd certainly consider that a target.

Principle of single responsibility would usually consider "do the thing and log it" to be a single action.

Reply:
Yeah, I was totally scratching my head at this when I first read it too. I don't know why it is the way it is but didn't want to create any more noise unrelated to my changes at this point haha.